src/components/BracketCard/BracketCard.styles.ts

```diff
@@ -61,6 +61,6 @@ export const textColors = {
 };
 export type TextColorType = keyof typeof textColors;
 
-export const tab = 'su-mt-[50vw] su-mb-26 su-h-14 lg:su-h-16 xl:su-h-20 2xl:su-h-[2.4rem] su-max-w-full su-w-200 xl:su-w-300 2xl:su-w-[35rem] sm:su-my-16 xl:su-my-36 2xl:su-my-48';
+export const tab = 'su-mt-[50vw] su-mb-26 su-h-14 lg:su-h-16 xl:su-h-20 2xl:su-h-24 su-max-w-full su-w-200 xl:su-w-300 2xl:su-w-[35rem] sm:su-my-16 xl:su-my-36 2xl:su-my-48';
```

Comment:
Added new spacing values in theme, so that any values between 1-30 doesn't need []